### ISSUE TITLE
Support running postgresql in a separate container

### ISSUE THREAD

**Issue by alexanderkjeldaas** [openlabs/tryton#13]
Hi, I think it would be better if this docker container supported running postgresql in a separate container and link them together.  I don't know tryton that well, but this makes it possible to manage data independently from the application, and this is typically the best way to setup docker.  For example a server could potentially reuse a postgresql database across multiple containers.

``` shell
docker run --name postgresql -e ROOT_PASSWORD=foo -d postgresql
docker run --name tryton --link postgresql:db -d tryton
```

**Comment by sharoonthomas** [comment]
@alexanderkjeldaas I agree with the concept and I think that is already the way it is. This Dockerfile does not install postgres. It could connect to the postgres server (another docker container, IP, domain name) specified in the `trytond.conf` file.

@prakashpp does trytond use `PG*` environment variables for connection if not specified in trytond.conf ?

**Comment by prakashpp** [comment]
@sharoonthomas trytond uses `TRYTOND_DATABASE_URI` variable to connect to db. We can use this uri to run tests on postgres.
